src/views/Summary.js

```diff
@@ -0,0 +1,31 @@
+import AddExpensesForm from '../components/AddExpensesForm';
+
+const Summary = () => {
+  return (
+    <main className="container-md">
```

Comment:
I am not sure if the main container should go into the summary. conceptionally, I would think that the main container could include more than just the summary (e.g. the main container would contain the summary _and_ the form).

Reply:
Indeed, I'll remove it. Please also see comment above about the potential use of a <Layout /> component

Reply:
general remark, can you please resolve the comment once you have made the change (or we discussed not to make a change)? I am currently not sure which comments are resolved already and which ones are not. I will also try to resolve comments as soon as I see them done.

Reply:
sure